data/Music/Sonic the Hedgehog Series/Sonic CD/jpbadstardustspeedway.meta

```diff
@@ -1,3 +1,4 @@
 Sonic CD (JP) - Stardust Speedway Bad Future
 0x03
-bgm
\ No newline at end of file
+bgm
+BigFight,DampeRace
```

Comment:
The location line should have at least one group from each level of specificity (exact, high, mid and low).